trade-finance-manager-ui/server/controllers/helpers/deal-cancellation-enabled.helper.ts

```diff
@@ -0,0 +1,6 @@
+import { DEAL_SUBMISSION_TYPE, DealSubmissionType, isTfmDealCancellationFeatureFlagEnabled } from '@ukef/dtfs2-common';
+
+export const dealCancellationEnabled = (submissionType: DealSubmissionType): boolean => {
+  // TODO: DTFS2-7298: also check that the deal hasn't already been cancelled.
+  return isTfmDealCancellationFeatureFlagEnabled() && (submissionType === DEAL_SUBMISSION_TYPE.AIN || submissionType === DEAL_SUBMISSION_TYPE.MIN);
```

Comment:
Minor point for clarity, may be deconstruct `DEAL_SUBMISSION_TYPE` so the above can be 

```typescript
const acceptableSubmissionType = submissionType === AIN || submissionType === MIN;

return isTfmDealCancellationFeatureFlagEnabled() && acceptableSubmissionType;
```

Reply:
Good idea, I've changed it to this, and I've also added a check here that the user is a PIM user

Reply:
Resolving as this has been addressed